.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name: conda-store-ui documentation
```

Comment:
From my understanding, there are shared docs for the JupyterLab extension and the conda-store-ui